src/algebra/direct_sum.lean

```diff
@@ -3,24 +3,24 @@ Copyright (c) 2019 Kenny Lau. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kenny Lau
 
-Direct sum of abelian groups, indexed by a discrete type.
+Direct sum of commutative monoids, indexed by a discrete type.
```

Comment:
```suggestion
Direct sum of additive commutative monoids, indexed by a discrete type.
```
No mathematical content, of course, but just to be Lean-precise.

Reply:
Please move this line to a module doc string.

Reply:
> > Please move this line to a module doc string.
> 
> I'd rather not touch the docstring more the necessary in this PR, else it will conflict with #3473

Ah, I thought this looked familiar...